Sources/CacheAdvance/BigEndianHostSwappable.swift

```diff
@@ -31,10 +31,10 @@ extension BigEndianHostSwappable {
     ///
     /// - Parameter data: A data blob representing encodable data. Must be of length `Self.storageLength`.
     init(_ data: Data) {
-        let decodedSize = withUnsafePointer(to: data) {
-            return UnsafeRawBufferPointer(start: $0, count: Self.storageLength)
+        let decodedSize = data.withUnsafeBytes {
+            return $0.load(as: Self.self)
         }
-        self = Self.swapToHost(decodedSize.load(as: Self.self))
+        self = Self.swapToHost(decodedSize)
```

Comment:
Woof. I don't understand why this change was necessary to get CI passing in Release on Xcode 13 – Debug worked fine.

The good news is the only tests that were failing were:
```
-[CacheAdvanceTests.BigEndianHostSwappableTests test_init_canBeInitializedFromEncodedData]
-[CacheAdvanceTests.EncodableMessageTests test_encodedData_encodesCorrectSize]
```

These tests are effectively testing implementation details – the integration tests where we persist messages and de-persist them were still succeeding.

So, what does this change do? Instead of creating a buffer pointer from the data's pointer, I instead access the underlying data's bytes directly. Skipping the translation to a buffer pointer simplifies what's happening here, which fixed the underlying issue. Skipping the translation buffer likely also gave us a very small performance win.